packages/react-union-scripts/scripts/webpack/loaders.parts.js

```diff
@@ -34,20 +12,22 @@ const loadBabel = include => () => ({
 	},
 });
 
-const loadScss = include => debug => ({
+const loadCss = include => (sourceMap, isServerConfig) => ({
 	module: {
 		rules: [
 			{
-				test: /\.scss$/,
+				test: /\.s?css$/,
```

Comment:
we are currently dropping Sass support, check it after it will be merged

Reply:
Yup, certainly